cmd/minishift/cmd/start.go

```diff
@@ -605,9 +605,6 @@ func initClusterUpFlags() *flag.FlagSet {
 	clusterUpFlagSet.AddFlag(cmdUtil.HttpsProxyFlag)
 
 	if minishiftConfig.EnableExperimental {
-		clusterUpFlagSet.Bool(configCmd.Metrics.Name, false, "Install metrics (experimental)")
-		clusterUpFlagSet.Bool(configCmd.Logging.Name, false, "Install logging (experimental)")
-		clusterUpFlagSet.Bool(configCmd.ServiceCatalog.Name, false, "Install service catalog (experimental)")
```

Comment:
We also need to update the integration test case as well, as of now failing for CI.

Reply:
addressed